app/static/openapi.yaml

```diff
@@ -210,21 +210,16 @@ paths:
                       name: 'Docker'
                     - id: 15
                       name: 'Regular Expressions'
-                    - id: 1
+                    - id: 16
                       name: 'Mentorship'
-                    - id: 1
+                    - id: 17
                       name: 'News'
-                    - id: 1
+                    - id: 18
                       name: 'Back End Dev'
-                    - id: 1
+                    - id: 19
                       name: 'Mobile Dev'
-                    - id: 1
+                    - id: 20
                       name: 'Cheat Sheets'
-                  has_next: true
-                  has_prev: false
-                  number_of_pages: 4
-                  page: 1
-                  records_per_page: 20
                   status: 'ok'
                   status_code: 200
                   total_count: 72
```

Comment:
Similar issue here, let's make this 23 and add the 23 categories that are in the local db right now to the list above